managed/src/main/java/com/yugabyte/yw/models/InstanceType.java

```diff
@@ -185,6 +196,22 @@ public static void deleteInstanceTypesForProvider(Provider provider) {
       entry.getInstanceTypeCode())).collect(Collectors.toList());
   }
 
+  /**
+   * TODO: Please remove this method when we have helperMethod in place to get config details
+   * Query Helper to find supported instance types for a given cloud provider.
+   */
+  public static List<InstanceType> findByProvider(Provider provider, Config config) {
```

Comment:
Do not make separate method. Just update same method.

Reply:
@sb-yb  I thought of doing this way but we will have to update it every place like AppInit.java and other places wherever  this method is being used. 
We know this is temp fix, anyway we are going to refactor the code. That's why created overloaded method to avoid changing everywhere.

Reply:
Updated! Thank you!